src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -26,11 +28,22 @@
   "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" befindet sich bereits im Shaderpaket-Ordner!",
   "options.iris.shaders.disabled": "Shader: Deaktiviert",
   "options.iris.shaders.enabled": "Shader: Aktiviert",
+  "options.iris.back": "Zurück",
+  "options.iris.reset": "Zurücksetzen",
+  "options.iris.reset.tooltip": "Setze ALLE Einstellungen zurück und wende diese an?",
```

Comment:
This sounds a bit weird because you don't really have a subject in this question

Reply:
good point